spec/lib/catalog/workspace_builder_spec.rb

```diff
@@ -0,0 +1,59 @@
+describe Catalog::WorkspaceBuilder, :type => [:topology, :service] do
```

Comment:
Don't think you need the `:topology` type here since we're not making any calls to topology in this service, right?